src/Auth0Client.ts

```diff
@@ -693,14 +702,8 @@ export default class Auth0Client {
   private async _getTokenUsingRefreshToken(
     options: GetTokenSilentlyOptions
   ): Promise<any> {
-    options.scope = getUniqueScopes(
```

Comment:
This function returns `options.scope`, do you no longer want to overwrite it?

Reply:
I originally removed it because`getTokenSilently` already passes in the scope with the defaults, class options and func options mushed together, I figured doing it here as well was redundant. By the same token, returning the scopes out of here is probably redundant as well but does make the code a little simpler as we don't have to re-attach the scopes after calling `_getTokenUsingRefreshToken`.

https://github.com/auth0/auth0-spa-js/blob/703a6c341b5c42e205dbc458aa93d8cc3b971ed3/src/Auth0Client.ts#L502-L507

But then I'm mushing them together again on line 706 so this change does feel redundant anyway. I'll revert it and keep it the same.